src/sentry/api/serializers/models/organization_dashboard.py

```diff
@@ -53,7 +58,9 @@ class DashboardWithWidgetsSerializer(Serializer):
     def get_attrs(self, item_list, user):
         result = {}
 
-        widgets = serialize(list(Widget.objects.filter(dashboard_id__in=[i.id for i in item_list])))
+        widgets = serialize(
+            list(DashboardWidget.objects.filter(dashboard_id__in=[i.id for i in item_list]))
+        )
```

Comment:
It'll be nice to have this pre-sorted by `order` so that we don't have to sort it on the client-side.

Reply:
Good idea. I'll also remove the order field. We can use the order of the widgets as the 'order' and not need to maintain additional state on the client.